packages/front-end/pages/api/init.ts

```diff
@@ -53,6 +55,7 @@ export default function handler(req: NextApiRequest, res: NextApiResponse) {
   const body: EnvironmentInitValue = {
     appOrigin: APP_ORIGIN || "http://localhost:3000",
     apiHost: API_HOST || "http://localhost:3100",
+    cdnHost: CDN_HOST,
```

Comment:
You are typing it above as string, but this may be undefined.  You can do `CDN_HOST || ""` to force it to always be a string.